main_exercise/client/src/actions/todoList-actions.js

```diff
@@ -0,0 +1,42 @@
+import ItemClient from "../clients/item_client";
+
+
+const itemClient= new ItemClient()
+
+export const getTodoListAction = () => async (dispatch) => {
```

Comment:
```suggestion

const getTodoList = (todoList) => ({type: "GET_TODO_LIST", payload: todoList});

export const getTodoListAction = () => async (dispatch) => {
  const todoList = await itemClient.getTodoList()
  dispatch(getTodoList(todoList))
}
```

Do that for all the rest